CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ The next release will require at least [Go 1.22].
 - Add the `WithMetricsAttributesFn` option to allow setting dynamic, per-request metric attributes in `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`. (#5876)
 - The `go.opentelemetry.io/contrib/config` package supports configuring `with_resource_constant_labels` for the prometheus exporter. (#5890)
 - Support [Go 1.23]. (#6017)
+- Implement`http.Hijacker` in`go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux`. (#5402)
```

Comment:
```suggestion
- Implement `http.Hijacker` in`go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux`. (#5796)
```

Also, since we had a release, your changelog entry needs to be moved higher, to the `unreleased` section.